l10n_it_location_nuts/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# Copyright 2019 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'NUTS Regions for Italy',
+    'summary': 'NUTS specific options for Italy',
+    'version': '12.0.1.0.0',
+    'category': 'Localisation/Europe',
+    'website': 'https://www.agilebg.com',
```

Comment:
"name" e "summary" non sono traducibili, andrebbero in italiano e con "ITA -" davanti al nome del modulo. (vedi linee guida https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html)

"ITA - Regioni NUTS per l'Italia" o forse ancora meglio "ITA - Regioni NUTS", più compatto e non ripetitvo.
Opzioni NUTS specifiche per l'Italia

'website' andrebbe sostituito con il link al repository.

Reply:
fatto, grazie!

Reply:
Manca ancora questa (seconda riga) https://github.com/OCA/l10n-italy/pull/1471#discussion_r327498740